app/src/components/market/profile/closed_market_detail.tsx

```diff
@@ -140,6 +152,8 @@ export const ClosedMarketDetailWrapper = (props: Props) => {
     disabledColumns.push(OutcomeTableValue.Shares)
   }
 
+  const hasWinningOutcomes = payouts && payouts.some((payout, index) => payout > 0 && balances[index].shares.gt(0))
```

Comment:
Couldn't this also be `earnedCollateral.gt(0)`?

Reply:
Done, thank you!